api/src/main/java/com/okta/idx/sdk/api/response/AuthenticationResponse.java

```diff
@@ -43,6 +47,12 @@
 
     private ContextualData contextualData;
 
+    public CurrentAuthenticatorEnrollment currentAuthenticator;
+
+    public AuthenticatorEnrollments authenticatorEnrollments;
```

Comment:
This is only used internally, can we refactor to make this non public?

Reply:
refactored.